plugin/fulltext/mysql-test/ngram/ngram_missing_plugin.result

```diff
@@ -0,0 +1,23 @@
+SET NAMES utf8mb4;
+CREATE TABLE t (str TEXT, FULLTEXT(str) WITH PARSER ngram)
+ENGINE=InnoDB;
+INSERT INTO t VALUES ("Hello World"), ("abcdef ghijklm");
```

Comment:
Let us try to use the standard (single) quotes, unless there is a strong reason to do otherwise. In SQL, double quotes are supposed to be used around identifiers only. MySQL and MariaDB normally use backticks in that case.

Reply:
Oh. I've never thought about quotes and just assumed that both single and double quotes are allowed for strings, since it always worked just fine. Thank you for mentioning this.

Changed to single quotes.